packages/scandipwa/src/component/ProductGallery/ProductGallery.component.tsx

```diff
@@ -414,6 +414,30 @@ export class ProductGalleryComponent extends PureComponent<ProductGalleryCompone
         return url;
     }
 
+    renderPlaceholderImage(): ReactElement {
+        const { activeImage } = this.props;
+
+        const {
+            state: {
+                state: {
+                    product: { small_image: { url } },
+                },
+            },
+        } = history;
```

Comment:
probably it would be good to stay consistent and use `history` from the utils instead of the browser one
`import history from 'Util/History';`